### ISSUE TITLE
Implementing choropleth.

### ISSUE THREAD

**Issue by hifall** [xkjyeah/vue-google-maps#414]
Is it possible to use this component to implement something like a choropleth of the world?

Like this: http://bl.ocks.org/micahstubbs/8e15870eb432a21f0bc4d3d527b2d14f.

Thanks!

**Comment by xkjyeah** [comment]
Yes you can. Using polygons and the polygon object. You just need to find
the polygon definitions that trace the outlines of countries
